examples/python/wanderer_events_client.py

```
@@ -0,0 +1,501 @@
+    def handle_all_events(event):
+        print(f"📊 Stats: Received event {event.get('type')}")
+    
+    ws_client.on('add_system', handle_add_system)
+    ws_client.on('map_kill', handle_map_kill)
+    ws_client.on('*', handle_all_events)
+    
+    # Connect with specific event types and backfill
+    try:
+        # Last 5 minutes
+        since = datetime.now(timezone.utc).replace(microsecond=0).isoformat().replace('+00:00', 'Z')
+        since = (datetime.now(timezone.utc) - timedelta(minutes=5)).isoformat().replace('+00:00', 'Z')
```

Pylint (3.3.7)

[error] Possibly using variable 'timedelta' before assignment

(E0606)